src/Particle.js

```diff
@@ -1922,6 +1922,45 @@ class Particle {
 		});
 	}
 
+	/**
+	 * Get product device's configuration
+	 * @param  {Object} options          Options for this API call
+	 * @param  {String} options.product  Config for this product ID or slug
+	 * @param  {String} options.auth     Access Token
+	 * @param  {String} options.deviceId Device ID to access
+	 * @param {Object} [options.headers] Key/Value pairs like `{ 'X-FOO': 'foo', X-BAR: 'bar' }` to send as headers.
+	 * @param {Object} [options.context] Request context
+	 * @returns {Promise} A promise
```

Comment:
it's not your problem to solve, just calling it out: at some point we need to document what these promises resolve to - ideally, via stable entity definitions.